tools/analysis_tools/analyze_results.py

```diff
@@ -39,11 +43,24 @@ def bbox_map_eval(det_result, annotation):
     # mAP
     iou_thrs = np.linspace(
         .5, 0.95, int(np.round((0.95 - .5) / .05)) + 1, endpoint=True)
-    mean_aps = []
+
+    processes = []
+    workers = Pool(processes=nproc)
     for thr in iou_thrs:
-        mean_ap, _ = eval_map(
-            bbox_det_result, [annotation], iou_thr=thr, logger='silent')
-        mean_aps.append(mean_ap)
+        p = workers.apply_async(eval_map, (bbox_det_result, [annotation]), {
+            'iou_thr': thr,
+            'logger': 'silent',
+            'nproc': 1
+        })
+        processes.append(p)
+
+    workers.close()
+    workers.join()
+
+    mean_aps = []
+    for p in processes:
+        mean_aps.append(p.get()[0])
+
```

Comment:
Can use pool.starmap instead.

Reply:
Passing parameters is cumbersome for pool.starmap here.
The result of 'apply_async' is out of order but more faster than `apply`.